api/src/pcapi/sandboxes/scripts/creators/test_cases/__init__.py

```diff
@@ -170,27 +172,65 @@ def create_offer_with_ean(ean: str, venue: offerers_models.Venue, author: str) -
     offers_factories.StockFactory(quantity=random.randint(10, 100), offer=offer)
 
 
-def _create_offer_and_stocks_for_allocine_venues(venue: offerers_models.Venue) -> None:
-    film_offer = offers_factories.OfferFactory(
-        venue=venue,
-        subcategoryId=subcategories_v2.SEANCE_CINE.id,
-        name=f"Séance Allociné - {venue.name}",
-    )
-    for daydelta in range(30):
-        day = datetime.date.today() + datetime.timedelta(days=daydelta)
-        for hour in (0, 5, 11, 17, 21):
-            beginning_datetime = datetime.datetime.combine(day, datetime.time(hour=hour))
-            is_full = random.random() < 0.3
-            quantity = random.randint(1, 100) if not is_full else 0
-            offers_factories.StockFactory(
-                offer=film_offer,
-                beginningDatetime=beginning_datetime,
-                bookingLimitDatetime=beginning_datetime - datetime.timedelta(minutes=30),
-                quantity=quantity,
+def _create_offer_and_stocks_for_allocine_venues(
+    venues: list[offerers_models.Venue], products: list[offers_models.Product]
+) -> None:
+    for venue in venues:
+        for idx, product in enumerate(products):
+            movie_offer = offers_factories.OfferFactory(
+                durationMinutes=product.durationMinutes,
+                name=product.name,
+                product=product,
+                subcategoryId=subcategories_v2.SEANCE_CINE.id,
+                venue=venue,
+                extraData=product.extraData,
             )
+            product_stocks = []
+            for daydelta in range(30):
+                day = datetime.date.today() + datetime.timedelta(days=daydelta)
+                for hour in (0, 5, 11, 17, 21):
+                    beginning_datetime = datetime.datetime.combine(day, datetime.time(hour=hour))
+                    is_full = hour % 5
```

Comment:
```suggestion
                    is_full = hour > 5
```
ça retire le côté mystique et intimidant de `%`

Reply:
Oups, le `%` est simplement équivalent à un `==` en fait